app/javascript/articles/Feed.jsx

```diff
@@ -50,19 +52,20 @@ export class Feed extends Component {
    *
    * @returns {Promise} A promise containing the JSON response for the feed data.
    */
-  static getFeedItems(timeFrame = '', page = 1) {
-    return fetch(`/stories/feed/${timeFrame}?page=${page}`, {
+  const getFeedItems = async (timeFrame = '', page = 1) => {
```

Comment:
syntax (non-blocking): There is no need for these to be arrow functions, they can be regular functions.